README.md

```diff
@@ -7,3 +7,10 @@ System Design for Tiny Url
 3. Services/APIs
 4. Data Model/ Class Design
 5. Bottlenecks/Scaling
+
+Tech Stack
```

Comment:
Hi @pmventura , can you please change "Tech Stack" to "Tech Stack for Simple Version"